schemas/caching-schema.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://zowe.org/schema/api-layer-config",
+    "title": "Caching service configuration",
+    "description": "Properties that can be used to configure caching service in a configuration file, such as zowe.yaml.",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "port": {
+            "$ref": "#/$defs/port",
+            "default": 7555
+        },
+        "debug": {
+            "type": "boolean",
+            "description": "Enable debug logs in discovery service"
+        },
+        "storage": {
+            "evictionStrategy": {
+                "type": "string",
+                "description": "Strategy for caching entries eviction.",
+                "default": "reject"
+            },
+            "mode": {
+                "type": "string",
+                "description": "Type of storge in caching service.",
+                "enum": ["inMemory","VSAM","redis"],
+                "default": "inMemory"
+            },
+            "size": {
+                "type": "integer",
+                "description": "the number of records stored before the eviction strategy is initiated",
+                "default": 10000
+            },
+            "vsam": {
```

Comment:
Missing redis properties - I can add if you like?

Reply:
I think the issue here is that it will not be propagated to caching service, there are no parameters in start.sh related to redis configuration. Do you think it is worth updating the start script as a part of this PR?